src/lwc/geFormRenderer/geFormRenderer.js

```diff
@@ -1766,4 +1749,247 @@ export default class GeFormRenderer extends NavigationMixin(LightningElement) {
     get qaLocatorSaveButton() {
         return `button ${this.saveActionLabel}`;
     }
+
+    /*******************************************************************************
+     * @description Object used to hold current values for all fields on the form.
+     */
+    @track
+    _formState = {}
```

Comment:
Can we move the `_formState` variable to the top of the class since its a top-level variable?

Reply:
Yep I'll move it up!